### DISCUSSION TITLE
Hello sir

### DISCUSSION THREAD

**Discussion by developer-krushna** [REAndroid/ARSCLib#25] [Q&A]
Using your library we can patch or Modify an arsc file without decompile.. and we can understand it by seeing your given example..but sir may I know how can I use this library for patching a number of layout ids, view ids, string ids without decompiling the arsc file sir ..is this possible with your library...As I know its possible I think because your library can Modify the package name of arsc file..So thank you ♥️

And I'm waiting for your valuable reply 😊

**Comment by REAndroid** [comment]
Yes you can modify anything on arsc and binary xml. I know this lib lacks full documentation, but I can help you if you already started something
